libs/Texture/Texture.h

```diff
@@ -23,15 +23,17 @@
 #endif
 
 #include <string>
+#include <vector>
 
 class Texture
 {
 public:
 	Texture(GLenum TextureTarget, const std::string& filename);
 	~Texture();
 
-	void Bind(GLenum TextureUnit);
+	void Bind();
 	GLuint GetTextureId();
+	void UnBind();
```

Comment:
and these functions are title case, why?

Reply:
This is my personal library to load textures. It has different notation.